ansible/roles/master/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+kubernetes_version: v1.0.3
```

Comment:
Found some solution for get latest release:
* Remove `kubernetes_version` from defaults
* Add task:
```
- name: Determine latest release
  shell: /bin/curl https://github.com/kubernetes/kubernetes/releases/latest 2>/dev/null | /bin/perl -n -e'/(https.*)(v\d([^\"]*))/ && print $2'
  when: kubernetes_version is undefined
```
* Add (commented out) `kubernetes_version` to group_vars for those, who wants some exact release.

***

About a task:
```
curl https://github.com/kubernetes/kubernetes/releases/latest
```
will get something like 
```
<html><body>You are being <a href="https://github.com/kubernetes/kubernetes/releases/tag/v1.0.4">redirected</a>.body></html>
```
with is redirect to latest release.
And
```
/bin/perl -n -e'/(https.*)(v\d([^\"]*))/ && print $2'
```
just regex to extract version from curl job.
I would prefer to use Python for this (since it is guarantee be in ansible managed nodes), but not familiar with it =/